auth-server/src/utils/validateEmail.ts

```diff
@@ -1,5 +1,5 @@
 export default (email: string): boolean => {
-	const regexp = /^[A-Z0-9_'%=+!`#~$*?^{}&|-]+([\.][A-Z0-9_'%=+!`#~$*?^{}&|-]+)*@[A-Z0-9-]+(\.[A-Z0-9-]+)+$/i
+	const regexp = /^[A-Z0-9_'%=+!`#~$*?^{}&|-]+([\.][A-Z0-9_'%=+!`#~$*?^{}&|-]+)*@[A-Z0-9-]+(\.[A-Z0-9-]+)+$/i // eslint-disable-line
```

Comment:
I disagree to mute eslint on that, I tested it in regexr.com and the slash inside `[]` isn't needed as the point is understood as a point.

Reply:
done